moped-editor/src/views/projects/projectsListView/helpers.js

```diff
@@ -2,7 +2,7 @@ import { useMemo } from "react";
 import { NavLink as RouterLink, useLocation } from "react-router-dom";
 import Link from "@mui/material/Link";
 import parse from "html-react-parser";
-import { formatDateType, formatTimeStampTZType } from "src/utils/dateAndTime";
+import { formatDateType, FormattedDateString, formatTimeStampTZType } from "src/utils/dateAndTime";
```

Comment:
I'm seeing a compile warning that `formatTimeStampTZType` isn't being used. could you remove? 🙏